controller-server/src/main/java/com/yahoo/vespa/hosted/controller/ApplicationController.java

```diff
@@ -645,12 +619,13 @@ private void validate(DeploymentSpec deploymentSpec) {
                 });
     }
 
-    /** Verify that change is tested and that we aren't downgrading */
+    /** Verify that what we want to deploy is tested and that we aren't downgrading */
     private void validateChange(Application application, ZoneId zone, Version version) {
         if ( ! application.deploymentJobs().isDeployableTo(zone.environment(), application.change())) {
```

Comment:
This isn't in sync with what is deployed now, but we should never trigger things that haven't already passed this check at triggering-time. I will fix this when I add specific versions for system and staging tests, based on what will be deployed in production.

Reply:
Actually, we could trigger something through `screwdriver/v1 ` that wasn't tested yet. If we cancel the change between triggering, and deployment, this could go wrong. Of course, this won't happen, or at least not until I fix this :p